### PR TITLE
Move NXP USB File to zephyr/modules

### PR DESCRIPTION
**Pull request by mmahadevan108**

### CODE REVIEW

**Comment by galak** [comment]
@mmahadevan108 my changes are merged so you should be able to rebase this now.

**Comment by mmahadevan108** [comment]
> @mmahadevan108 my changes are merged so you should be able to rebase this now.

Awesome, rebased and updated this PR

**Review by dleach02 — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 1d823ea49bf7f6229a1097fbff33e27ba0dd389b
+      revision: pull/104/head
```
Comment (MaureenHelm) [inline west.yml:91]: The hal PR has been merged, so you can now update this to the new SHA

Comment (mmahadevan108) [reply west.yml]: Updated `west.yml`

**Comment by mmahadevan108** [comment]
I am not clear why I am seeing the below error, any suggestion?
```
--- trusted-firmware-m: fetching, need revision 1ab3b62d641070a6e8574634168c374645f382b8
From https://github.com/zephyrproject-rtos/trusted-firmware-m
 * [new branch]      master     -> refs/west/master
HEAD is now at 1ab3b62d trusted-firmware-m: platform: lpcxpresso55s69: check-out HAL dependencies
HEAD is now at 1ab3b62d trusted-firmware-m: platform: lpcxpresso55s69: check-out HAL dependencies
ERROR: update failed for project tensorflow
```

**Review by MaureenHelm — APPROVED** [review]

**Comment by mmahadevan108** [comment]
I have rebased to the latest to fix CI issues.

**Pull request merged**